manual_update_control.md

```diff
@@ -23,9 +23,11 @@ The new function `update_mode()` sets the default update mode for resources. It
 `k8s_resource` and `dc_resource` take an additional optional argument `update_mode` (default value: `UPDATE_AUTO`), which must be one of the above constants.
 
 ## UI Changes
+* On startup, resources with manual update_mode will be dirty and unbuilt (other than Tiltfile).
 * Pressing 'u' will start an update of the selected resource (including Tiltfile). (If there is a current update running, it will put it the selected resource at the front of the queue).
 * Pressing shift+'u' will do the same, unless the selected resource is using Live Update, in which case it will do an image update, not a live update.
-* Pressing spacebar will toggle Pause. While Pause is on, Tilt will not queue builds automatically. At the moment the user turns off Pause, all resources with pending file edits will be enqueued.
+* Pressing 'y' will start an update of all dirty resources.
+* Pressing spacebar will toggle Pause. While Pause is on, Tilt will not queue builds automatically (including Tiltfile). At the moment the user turns off Pause, all resources with update_mode auto that have pending file edits will be enqueued.
```

Comment:
To be clear, are
a. auto-update and
b. manual-update
resources still accumulating changed files while things are paused?